docs/docs/authentication-tutorial.md

```diff
@@ -103,6 +103,7 @@ const Layout = ({ children }) => (
             paddingTop: 0,
           }}
         >
+          // highlight-next-line
```

Comment:
```suggestion
          {/* highlight-next-line */}
```

(I'm not going to comment on _all_ of these, so if you're able, another look at the JSX snippets would probably be a good idea!)

Reply:
Alright, I'd do that

Reply:
If it helps, using VSCode (and possibly extensions in other editors) is really helpful as it will syntax highlight the snippets. So it's clear at a glance if a snippet is broken!

Reply:
Whoops - apparently not. Just need to spot check them!